### PR TITLE
Add whatis page

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

Create a What Is page for Podman based on prior customer comments on the site.

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@rhatdan @mheon PTAL

File: index.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 <h3>Welcome to the website for the Pod Manager tool (<a href="https://github.com/containers/libpod">podman</a>). This site features announcements and news around Podman, and occasionally other <a href="https://github.com/containers/">container tooling</a> news.</h3>
 
-<h3>What is Podman?  Simply put: `alias docker=podman`
+<h3>What is Podman?  Simply put: `alias docker=podman` and more details <a href="whatis.html">here</a>.
```
Comment (mheon) [inline index.html]: Phrasing seems a little awkward. Maybe put this in parens, or on a new line?

File: whatis.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+At a high level, the scope of libpod and Podman is the following:
+
+<ul>
+<li>Support multiple image formats including the OCI and Docker image formats.</li>
+<li>Support for multiple means to download images including trust & image verification.</li>
+<li>Container image management (managing image layers, overlay filesystems, etc).</li>
+<li>Full management of container lifecycle.</li>
+<li>Support for pods to manage groups of containers together.</li>
+<li>Resource isolation of containers and pods.</li>
+<li>Integration with CRI-O to share containers and backend code.</li>
```
Comment (mheon) [inline whatis.html]: Let's not put this one on here until we actually do it

File: whatis.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+</ul>
+
+Podman is an open-source projet that is available on most Linux platforms and
```
Comment (mheon) [inline whatis.html]: Projet -> Project

File: whatis.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+<li>Support for multiple means to download images including trust & image verification.</li>
+<li>Container image management (managing image layers, overlay filesystems, etc).</li>
+<li>Full management of container lifecycle.</li>
+<li>Support for pods to manage groups of containers together.</li>
+<li>Resource isolation of containers and pods.</li>
+<li>Integration with CRI-O to share containers and backend code.</li>
+</ul>
+
+Podman is an open-source projet that is available on most Linux platforms and 
+resides here in <a href="https://github.com/containers/libpod">GitHub</a>.
```
Comment (mheon) [inline whatis.html]: Maybe just "resides at <link>Github</link>

File: whatis.html [hunk 5]
```diff
@@ -0,0 +1,64 @@
+<head>
+<link rel="shortcut icon" type="image/x-icon" href="/images/favicon.ico">
+</head>
+
+<img src="images/podman.svg" alt="podman logo">
+
+<h2>What is Podman?  Simply put: `alias docker=podman`</h2>
+
+Podman manages pods, containers, container images, and container volumes using the 
+<a href="https://github.com/containers/libpod">libpod</a> library.
```
Comment (mheon) [inline whatis.html]: This probably needs more detail - "Podman is a daemonless container engine that provides a Docker-compatible command line frontend", etc

File: whatis.html [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+<h3>Overview and scope</h3>
+
+At a high level, the scope of libpod and Podman is the following:
+
+<ul>
+<li>Support multiple image formats including the OCI and Docker image formats.</li>
+<li>Support for multiple means to download images including trust & image verification.</li>
```
Comment (mheon) [inline whatis.html]: I don't know what this really means... Should either explain better or drop

File: whatis.html [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+<li>Container image management (managing image layers, overlay filesystems, etc).</li>
+<li>Full management of container lifecycle.</li>
+<li>Support for pods to manage groups of containers together.</li>
+<li>Resource isolation of containers and pods.</li>
```
Comment (mheon) [inline whatis.html:23]: Do we need to call this out explicitly? I think this is implied by the pod/container concept

File: whatis.html [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+</ul>
+
+Podman is an open-source projet that is available on most Linux platforms and 
+resides here in <a href="https://github.com/containers/libpod">GitHub</a>.
+Podman is a command line tool that works on Open Container Initiative (OCI)
+images and containers.
+
+Podman specializes in all of the commands and functions that help you to maintain and modify
+OCI images, such as pulling and tagging.  It also allows you to create, run, and maintain those containers
+created from those images in a production environment.  Podman uses familiar container
```
Comment (mheon) [inline whatis.html]: Let's just explicitly say Docker compatability here, instead of "familiar commands"

File: whatis.html [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+<li>Container image management (managing image layers, overlay filesystems, etc).</li>
+<li>Full management of container lifecycle.</li>
+<li>Support for pods to manage groups of containers together.</li>
+<li>Resource isolation of containers and pods.</li>
+<li>Integration with CRI-O to share containers and backend code.</li>
+</ul>
+
+Podman is an open-source projet that is available on most Linux platforms and 
+resides here in <a href="https://github.com/containers/libpod">GitHub</a>.
+Podman is a command line tool that works on Open Container Initiative (OCI)
```
Comment (mheon) [inline whatis.html]: Should roll this paragraph and the next one up into one and put it at the top, right after "what is Podman"

File: index.html (hunk 1 above)
Comment (TomSweeneyRedHat) [reply index.html]: I stole a bit from @rhatdan's comment in a related issue and merged this thought in too.

File: whatis.html (hunk 5 above)
Comment (TomSweeneyRedHat) [reply whatis.html]: I've tried reworking, please re-review.

File: whatis.html (hunk 6 above)
Comment (TomSweeneyRedHat) [reply whatis.html]: FWIW, I grabbed this off the README.md

File: whatis.html (hunk 7 above)
Comment (TomSweeneyRedHat) [reply whatis.html:23]: I think we should, I think it's a good point to reiterate.

File: whatis.html (hunk 6 above)
Comment (TomSweeneyRedHat) [reply whatis.html]: I've add "to securely download", I think that clears it up.

File: whatis.html (hunk 2 above)
Comment (TomSweeneyRedHat) [reply whatis.html]: This too is on the README.md,  I've removed from here, but do we need/want to adjust there too?

File: whatis.html (hunk 3 above)
Comment (TomSweeneyRedHat) [reply whatis.html]: fixed, ty

File: whatis.html (hunk 4 above)
Comment (TomSweeneyRedHat) [reply whatis.html]: I went with "resides on"

File: whatis.html (hunk 8 above)
Comment (TomSweeneyRedHat) [reply whatis.html]: fixed

File: whatis.html (hunk 9 above)
Comment (TomSweeneyRedHat) [reply whatis.html]: I've removed from here and transmorgified it with some other stuff.

**Comment by TomSweeneyRedHat** [comment]
I've pushed a new variant.

**Comment by rhatdan** [comment]
I am going to merge, we can fix other issues in future PRs.

**Pull request merged**